vision_agent/agent/vision_agent.py

```diff
@@ -581,7 +654,10 @@ def chat_with_workflow(
 
             int_chat = cast(
                 List[Message],
-                [{"role": c["role"], "content": c["content"]} for c in chat],
+                [
+                    {"role": c["role"], "content": c["content"], "media": c["media"]}
```

Comment:
Should we resize media ? Not sure how many tokens that would be if the image resolution is too large

Reply:
For low res it always resizes to 512x512 https://platform.openai.com/docs/guides/vision but we should look into high res at some point